4_arm.ipynb

```diff
@@ -185,7 +326,20 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "# Motion generation using inverse kinematics\n",
+    "\n",
+    "You have probably understood that it is unexpectedly difficult to set joint values of the arm to make a desired movement. By using \"Inverse Kinematics\", we can set the angle of the joints automatically by setting the target position of the hand."
```

Comment:
"You probably understand that it is unexpectedly difficult to set joint values of the arm to make a desired movement. By using \"Inverse Kinematics\", we can set the angle of the joints automatically by setting the target position of the hand."